### DISCUSSION TITLE
error: failed to connect to Bitcoin Core RPC at 127.0.0.1:8332/wallet/ord because: I/O error: The system cannot find the file specified. (os error 2)

### DISCUSSION THREAD

**Discussion by Just-JB** [ordinals/ord#1607] [Q&A]
when trying to run "ord wallet create". noob question but I can't figure out a way around this error. I've enabled the RPC in bitcoin core. Tried running ord in the Bitcoin directory...

**Comment by casey** [comment]
This sounds like a missing cookie file! First step is to check for `.cookie`.

**Reply by bdochosiete** [reply]
I'm trying to create my ord wallet, I'm currently running my bitcoin node from an external SSD. I'm trying to set my data directory and cookie file directory to a different path  with `ord --cookie-file /path/to/my/.cookie` but nothing happens, I only get the `ord -h` response. What I'm I doing wrong?  I'm using Ubuntu.

**Reply by bdochosiete** [reply]
Oh nvm, looks like I have to use the sub commands, `ord --cookie-file /path/to/my/.cookie wallet create`. Is there a way to keep my actual path to the external SSD or do I have to specify the path with every command?
